content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+अगर आप **कर के** सीखना पसंद करते हैं तो हमारा [प्रैक्टिकल टुटोरिअल देखें](/tutorial/tutorial.html). इस टुटोरिअल में हम एक टिक-टैक्-टो का गेम बनाते हैं | हो सकता है आपको ऐसा लगे की हमें ये छोड़ देना चाहिए क्योकि हम कोई गेम नहीं बना रहे -- लेकिन इसे एक मौका दें | इसमें सीखी गई विधियां *किसी भी* React ऐप को बनाने में काम आएगी, और इसका अच्छा अभ्यास आपको React की गहरी समझ देगा | 
 
-### Step-by-Step Guide {#step-by-step-guide}
+### स्टेप-बाय-स्टेप गाइड {#step-by-step-guide}
 
-If you prefer to **learn concepts step by step,** our [guide to main concepts](/docs/hello-world.html) is the best place to start. Every next chapter in it builds on the knowledge introduced in the previous chapters so you won't miss anything as you go along.
+अगर आप React के **कांसेप्ट को स्टेप-बाय-स्टेप सीखना** पसंद करते हैं तो हमारी [मुख्य कॉन्सेप्ट्स की गाइड](/docs/hello-world.html) शुरू करने की लिए एक उत्तम जगह है | इसमें हर अगला अध्याय पिछले अध्यायों पर बना है तो इससे आप से कुछ छूटेगा नहीं |
 
-### Thinking in React {#thinking-in-react}
+### React में सोचें {#thinking-in-react}
 
-Many React users credit reading [Thinking in React](/docs/thinking-in-react.html) as the moment React finally "clicked" for them. It's probably the oldest React walkthrough but it's still just as relevant.
+बहुत सारे React के यूज़र्स [React में सोचे](/docs/thinking-in-react.html) पढ़ने को उनकी React कि समझ बढ़ाने में मदद करने का क्रेडिट देते हैं, वह समझते हैं कि इसको पढ़ने के बाद हि उन्हें React समझ में आई | ये React के सबसे पुराने वॉकथ्रू में से एक है | 
 
-### Recommended Courses {#recommended-courses}
+### रेकमेंडेड कोर्सेज {#recommended-courses}
 
-Sometimes people find third-party books and video courses more helpful than the official documentation. We maintain [a list of commonly recommended resources](/community/courses.html), some of which are free.
+कभी-कभी लोगों को अन्य पार्टियों कि किताबें और वीडियो कोर्सेज़ ऑफिसियल दस्तावेज़ों से ज़्यादा मददगार लगती है | हम एक [आम तौर पर इस्तेमाल करे जाने वाले साधनो की  सूचि](/community/courses.html) बनाए रखते हैं, जिनमे से कुछ मुफ्त हैं |
 
-### Advanced Concepts {#advanced-concepts}
+### एडवांस्ड कॉन्सेप्ट्स {#advanced-concepts}
 
-Once you're comfortable with the [main concepts](#main-concepts) and played with React a little bit, you might be interested in more advanced topics. This section will introduce you to the powerful, but less commonly used React features like [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
+जब आप [मुख्य कन्सेप्टों](#main-concepts) से आरामदायक हो जाएं और React का थोड़ा अभ्यास करलें हो सकता है आप एडवांस्ड विषयों में दिलचस्प हों | यह भाग आपको ताकतवर लेकिन कम इस्तेमाल किये जाने वाली [context](/docs/context.html) और [refs](/docs/refs-and-the-dom.html) जैसी React की खासियतों से आपका परिचय करवाएगा  |
```

Comment:
`यह भाग आपको ताकतवर लेकिन कम इस्तेमाल किये जाने वाली [context](/docs/context.html) और [refs](/docs/refs-and-the-dom.html) जैसी React की खासियतों से आपका परिचय करवाएगा  |` 

=> 

`यह भाग आपको ताकतवर लेकिन कम इस्तेमाल किये जाने वाली React की खासियत जैसे [context](/docs/context.html) और [refs](/docs/refs-and-the-dom.html) से आपका परिचय करवाएगा।`